src/utils.ts

```diff
@@ -209,9 +209,14 @@ export const getPRTitleComment = (storyTitle: string, prTitle: string): string =
     </p>
     `;
   }
-  return `<p>I'm a bot and I 👍 this PR title. 🤖</p>
 
-  <img src="https://media.giphy.com/media/XreQmk7ETCak0/giphy.gif" width="400" />`;
+  let s = `<p>I'm a bot and I 👍 this PR title. 🤖</p>`;
```

Comment:
minor nitpick. We would prefer meaningful variable names like `message`, `body`, `title`, etc instead of characters.

Reply:
Noted, will update this